Dockerfile

```diff
@@ -1,5 +1,11 @@
 FROM snapcore/snapcraft@sha256:6d771575c134569e28a590f173f7efae8bf7f4d1746ad8a474c98e02f4a3f627
 
+# Both upper and lower case variables are needed due to inconsistencies in curl, wget, etc
+ENV HTTP_PROXY=""
+ENV HTTPS_PROXY=""
+ENV http_proxy=""
+ENV https_proxy=""
```

Comment:
Found out after more testing: we can get away with only the lower case ones.

Reply:
Changed it to just lowercase, added examples